src/main/java/com/teradata/tpcds/ScalingInfo.java

```diff
@@ -69,21 +69,41 @@ public long getRowCountForScale(int scale)
         }
     }
 
-    private long computeCountUsingStaticScale()
+    private long computeCountUsingStaticScale(float scale)
     {
-        return getRowCountForScale(1);
+        if (scale == Math.round(scale)) {
+            return getRowCountForScale(1);
+        }
+        // Handle decimal scales (not part of spec). Will only have an effect on
+        // decimal scales 0 < x < 1.
```

Comment:
Dain said it's fine if the 0.1 scale generates ~180 MB. In the TPC-H generator they leave static tables as static.